src/backend/legendary/games.ts

```diff
@@ -615,6 +615,12 @@ class LegendaryGame extends Game {
    * Does NOT check for online connectivity.
    */
   public async syncSaves(arg: string, path: string): Promise<string> {
+    if (!path) {
+      logError('No path provided for SavesSync, check your settings!', {
+        prefix: LogPrefix.Legendary
+      })
+      return 'No path provided.'
```

Comment:
is this string displayed in the frontend? if so it should be translatable

Reply:
Only backend. But I guess this will never happens because of the checks I added on frontend as well.